docs/privacy-policy.rst

```diff
@@ -0,0 +1,361 @@
+
+We also collect potentially personally-identifying information
+like Internet Protocol (IP) addresses.
+
+
+Why do we collect this?
++++++++++++++++++++++++
+
+We collect this information to better understand how our website visitors
+use Read the Docs, and to monitor and protect the security of the website.
+
+
+Information from users with accounts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you **create an account**, we require some basic information at the time of account creation.
+You will create your own user name and password, and we will ask you for a valid email account.
+You also have the option to give us more information if you want to,
+and this may include "User Personal Information."
+
+"User Personal Information" is any information about one of our users which could,
+alone or together with other information, personally identify him or her.
+Information such as a user name and password, an email address,
+a real name, and a photograph are examples of "User Personal Information."
+
+User Personal Information does not include aggregated, non-personally identifying information.
+We may use aggregated, non-personally identifying information to operate, improve,
+and optimize our website and service.
+
+Why do we collect this?
++++++++++++++++++++++++
+
+- We need your User Personal Information to create your account, and to provide the services you request.
+- We use your User Personal Information, specifically your user name, to identify you on Read the Docs.
+- We use it to fill out your profile and share that profile with other users.
+- We will use your email address to communicate with you but it is not shared publicly.
+- We limit our use of your User Personal Information to the purposes listed in this Privacy Statement.
+  If we need to use your User Personal Information for other purposes, we will ask your permission first.
+  You can always see what information we have in your
+  `user account <https://readthedocs.org/accounts/edit/>`_.
+
+What information Read the Docs does not collect
+-----------------------------------------------
+
+We do not intentionally collect **sensitive personal information**,
+such as social security numbers, genetic data, health information, or religious information.
+
+Documentation Sites hosted on Read the Docs are public,
+anyone (including us) may view their contents.
+If you have included private or sensitive information in your Documentation Site,
+such as email addresses, that information may be indexed by search engines or used by third parties.
+
+If you're a **child under the age of 13**, you may not have an account on Read the Docs.
+Read the Docs does not knowingly collect information from or direct any of our content specifically to children under 13.
+If we learn or have reason to suspect that you are a user who is under the age of 13, we will unfortunately have to close your account.
+We don't want to discourage you from writing software documentation, but those are the rules.
```

Comment:
I'm not sure if we do it here in the docs, on signup, or both, but we should probably describe *why* this is.

At least, this last sentence seems harsh. "Those are the rules" could explain that because of GDPR and the information we collect, we can't allow signups from young users.

Reply:
I believe the GDPR requires permission if somebody is under 16! The US generally requires 13. This was essentially copied from GitHub's.